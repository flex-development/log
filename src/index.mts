/**
 * @file Package Entry Point
 * @module log
 */

export type * from '#interfaces/index'
export { default, default as logger } from '#logger'
export type * from '#types/index'
export * from '@flex-development/log/core'
export * from '@flex-development/log/reporters'
