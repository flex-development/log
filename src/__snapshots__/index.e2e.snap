// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:log > should expose public api 1`] = `
[
  "default",
  "logger",
  "logLevels",
  "createLogger",
  "Reporter",
  "BaseReporter",
  "FancyReporter",
]
`;
